code/game/objects/items/devices/radio/radio.dm

```diff
@@ -14,7 +14,7 @@ var/global/list/default_internal_channels = list(
 	num2text(SCI_FREQ) = list(access_tox, access_robotics, access_xenobiology),
 	num2text(SUP_FREQ) = list(access_cargo, access_mining_station),
 	num2text(SRV_FREQ) = list(access_janitor, access_library, access_hydroponics, access_bar, access_kitchen),
-	num2text(EXP_FREQ) = list(access_explorer, access_pilot)
+	num2text(EXP_FREQ) = list(access_awayteam)	//VOREStation Edit

```

Comment:
IMO, pilots could keep access to the channel, but that's more opinion suggestion

Reply:
I don't think this was actually intended. Pilots lost their explo radio key way back when they got separated from explo. This just let them access it through shortwaves and intercoms.

I don't think it's going to be all that relevant unless there's someone handing out away team headsets/keys anyway, in which case I figure they can just also ask

Reply:
Makes sense.